test/spec/geo/map.spec.js

```diff
@@ -191,6 +191,21 @@ describe('core/geo/map', function () {
     ]);
   });
 
+  describe('Layer action methods', function () {
+    describe('.moveCartoDBLayer', function () {
+      it('should provide a way to move CartoDB layers', function () {
+        expect(this.map.moveCartoDBLayer).toBeDefined();
+      });
+
+      it('should trigger an event in order to notify other elements', function () {
+        spyOn(this.map, 'trigger');
```

Comment:
Instead of spying on `trigger`, we can bind an event and ensure the callback is invoked:

```
var moveCartoDBLayer = jasmine.createSpy('moveCartoDBLayer');
this.map.on('moveCartoDBLayer', callback);

this.map.moveCartoDBLayer(1, 2);

expect(moveCartoDBLayer).toHaveBeenCalledWith('wadus');
```